src/index.scss

```diff
@@ -30,346 +37,268 @@ $mobile-width: 750px;
 	@return m#{a}x(#{$numbers});
 }
 
-a:hover {
-	text-decoration: none;
-}
-body {
-	margin: 0;
-	font-family: Poppins, sans-serif;
-}
-html, body, #root, #app, .full-height {
-	height: 100%;
-}
-
 .desktop-hide {
 	@include desktop {
 		display: none;
 	}
 }
 .mobile-hide {
-	@include tablet {
+	@include mobile {
 		display: none;
 	}
 }
-
 .flex-container {
 	display: flex;
 	align-items: center;
 	flex-shrink: 0;
 }
 
-.desktop-banner {
-	width: 100%;
-
-	padding: 12px 0px;
-
-	font-size: 14px;
-	font-weight: 300;
-	text-align: center;
+a:hover {
+	text-decoration: none;
+}
+body {
+	margin: 0;
+	padding: 0;
 
-	background-color: $blue;
+	font-family: Poppins, sans-serif;
+	font-size: 2vw;
 	color: white;
-
-	span {
-		font-weight: 400;
-	}
 }
-
-#mlh-badge {
-	position: absolute;
-	height: 175px;
-	margin-left: 50px;
-
-	@include tablet {
-		height: 150px;
-		margin-left: 40px;
-	}
-
-	@include mobile {
-		/*68 px is the current height of the nav bar
-		will think of a better way to structure this later*/
-		width: default;
-		height: max(68px, calc(35px + 10vh));
-		margin-left: 15px;
-	}
+html, body, #root, #app {
+	height: 100%;
 }
 
-.navigation {
-	width: 100%;
+nav {
+	position: fixed;
+	z-index: 1;
 
-	font-size: 15px;
-	font-weight: 500;
+	width: 100vw;
+	height: 5vw;
 
-	color: $blue;
-	background-color: white;
+	background-color: transparentize(white, 0.95);
+	
+	#logo {
+		content:url('./bh7-assets/logo.svg');
+		height: 100%;
+		margin: 0vw 2vw;
+	}
 
 	ul {
-		list-style-type: none;
-		float: right;
-
-		margin: 0;
-		padding: 0;
-
-		li {
-			padding: 25px 25px;
-			text-align: center;
-			text-decoration: none;
+		display: flex;
+		align-items: center;
 
-			a {
-				color: $blue;
+		list-style-type: none;
 
-				border-bottom-style: solid;
-				border-color: transparentize($blue, 1);
-				transition: border-color 0.2s;
+		height: 100%;
+		padding-inline-start: 0;
 
-				&:hover {
-					border-color: transparentize($blue, 0);
-					transition: border-color 0.2s;
-				}
-			}
-		}
-	}
+		a {
+			display: flex;
+			align-items: center;
 
-	@include tablet {
-		display: inline-block;
-		
-		font-size: 12px;
-		font-weight: 400;
+			height: 100%;
+			margin: 0vw 2vw;
 
-		color: white;
-		background-color: $blue;
+			color: white;
 
-		ul li {
-			padding: 25px 15px;
-			a {
-				color: white;
-				border-bottom-style: solid;
-				border-color: transparentize(white, 1);
+			font-size: 1.25vw;
+			font-weight: 400;
 
-				&:hover {
-					border-color: transparentize(white, 1);
-				}
-				&:active {
-					border-color: transparentize(white, 0);
-				}
+			transition: font-weight 0.1s, color 0.2s, border-color 0.3s;
```

Comment:
Huh, didn't know this shorthand was a thing! 😄

Reply:
I'm also not a huuuge fan of how when you hover, the bold pushes the text around; I'll bring up at design sync if they mind if we just go all bold 😅